### ISSUE TITLE
Override parent module behavior?

### ISSUE THREAD

**Issue by jackfirth** [AlexKnauth/multi-file-lang#2]
By default, running a `#lang multi-file` module runs all declared file modules. Would it make sense for a special `#file self` block to define what happens when the multi-file module file is required?

My use case is a multi-file module that defines a file with test data and the parent module defines a test submodule that reads the data in a test. This way the existence of the test data file is completely encapsulated, `raco test` just runs the test submodule of the parent module like it would for any other module.

**Comment by AlexKnauth** [comment]
Okay. So the problem with the current version is that `raco test` doesn't see a `test` submodule in the `#lang multi-file` module, so it tries to run the whole thing.

**Comment by jackfirth** [comment]
Right. In general I'd like control over what the multi-file module actually does with each "subfile".
